ir-sanguinem-et-lacrimis/common/technologies/artillery.txt

```diff
@@ -1883,11 +1885,6 @@ technologies = {
 			position = { x = @at_missile_row y = @1951 }
 		}
 
-		path = {
-			leads_to_tech = at_rocket3
-			research_cost_coeff = 1
-		}
```

Comment:
has nothing to do with the speed issue; will include this in the bureau_and_tactics pull